contrib/kind.sh

```diff
@@ -106,6 +120,12 @@ SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/12}
 NET_CIDR_IPV6=${NET_CIDR_IPV6:-fd00:10:244::/48}
 SVC_CIDR_IPV6=${SVC_CIDR_IPV6:-fd00:10:96::/64}
 
+KIND_NUM_MASTER=1
+if [ "$KIND_HA" == true ]; then
+  KIND_NUM_MASTER=3
+  KIND_NUM_WORKER=0
```

Comment:
What I meant here was we only want to default KIND_NUM_WORKER to 0, if it is unset. Meaning that, a user could want to deploy HA with a lot of worker nodes (although that is not the default). So I think you can move line 110 and do:

```
if [ "$KIND_HA" == true ]; then
  KIND_NUM_WORKER=${KIND_NUM_WORKER:-0}
else
  KIND_NUM_WORKER=${KIND_NUM_WORKER:-2}
fi
```

Reply:
So you want to allow KIND_NUM_WORKER to be updated in HA and noHA. Miss that point. No problem, but I'll need to change the helptext to remove "Ignored in HA mode." and update the J2 to move the worker provisioning out of the ELSE:
  {%- if ovn_ha is equalto "true" %}
  {%- else %}
    <worker setup>
  {%- endfor %}